packages/block-library/src/block/edit.js

```diff
@@ -216,19 +218,23 @@ function ReusableBlockEdit( {
 
 	// Sync the editing mode of the pattern block with the inner blocks.
 	useEffect( () => {
-		setBlockEditMode(
-			setBlockEditingMode,
-			innerBlocks,
-			// Disable editing if the pattern itself is disabled.
-			editingMode === 'disabled' || ! hasPatternOverridesSource
-				? 'disabled'
-				: undefined
-		);
+		registry.batch( () => {
```

Comment:
I figured this should use `registry.batch`? Was it originally an oversight or is it unnecessary?